tasks/dataverse-prereqs.yml

```
@@ -46,16 +46,23 @@
     name: ['bash-completion', 'git', 'jq', 'mlocate', 'net-tools', 'sudo', 'unzip', 'python3-psycopg2', 'zip', 'tar']
     state: latest
 
-- name: "RHEL/Rocky 8.6 provides Ansible 5.4 which wants Python-3.8"
+- name: "RHEL/Rocky 8.6 packaged Ansible wants Python-3.8"
```

```suggestion
- name: "RHEL/Rocky 8.6-packaged Ansible wants Python-3.8"
```

... or something to make this a bit more clear. The old "provides" and "which" actually reads better to me.

What I think we're saying:

The version of Ansible provided by RHEL/Rocky 8.6 requires Python 3.8.

I'm not sure if I have this right though. 😅